contrib/platform/src/com/sun/jna/platform/win32/Advapi32Util.java

```diff
@@ -214,7 +214,6 @@ public static Account getAccountByName(String systemName, String accountName) {
 
 		Account account = new Account();
 		account.accountType = peUse.getPointer().getInt(0);
-		account.name = accountName;
```

Comment:
At first glance, this lines change appears to be more than a "redundant" assignment. Was it intentional?

Reply:
Look a few lines below, it’s overwritten in the if-elseif-else.

Reply:
Aha. I see it now: https://github.com/java-native-access/jna/pull/1009/files#diff-791566375b133e0e17cf2a68da8a7465R221